packages/design-system/src/components/Table/TableBody.jsx

```diff
@@ -15,7 +15,13 @@ export const TableBody = ({ children, _stackable, ...tableBodyProps }) => {
     });
   };
 
-  return <tbody {...tableBodyProps}>{_stackable ? renderChildren() : children}</tbody>;
+  /* eslint-disable jsx-a11y/no-redundant-roles */
+  return (
+    <tbody role="rowgroup" {...tableBodyProps}>
```

Comment:
Should we remove `rowgroup`?  
@davidakennedy mentioned it doesn't add any value so is there another reason to keep it?
> and adding the role="rowgroup" doesn't harm anything, but doesn't seem to improve anything either.

Reply:
My thought would be to leave it since screen readers/assistive tech may do something with it in the future. That said, I don't have strong feelings, so okay if you all want to remove them.